src/ir_Hitachi.h

```diff
@@ -41,6 +41,16 @@ const uint8_t kHitachiAcPowerOffset = 0;
 const uint8_t kHitachiAcSwingOffset = 7;
 
 // HitachiAc424
+// Byte[11]
+const uint8_t kHitachiAc424ButtonByte = 11;
+const uint8_t kHitachiAc424ButtonPower = 0x13;
+// TODO(jamsinclair): Get correct value for Mode Button. Currently same as Power
```

Comment:
The AC doesn't have one power button. By pressing one of the modes the AC will either turn on, or switch mode. Their is also an off button, they all share the same button byte value. 

I suppose we could just use `kHitachiAc424ButtonPower` for all of these?

Reply:
Sounds good to me. Perhaps rename the constant to `kHitachiAc424ButtonMode/Power`
Also adjust the text for it in `::toString()` .. `result += kPowerStr + '/' + kModeStr;`